src/components/DiveLog/DiveLog.js

```diff
@@ -34,7 +36,7 @@ export default function DiveLog({
           <BoxDateDelete>
             <Date>{date}</Date>
             <ButtonDelete>
-              <FaTrash onClick={() => onDelete(_id)} />
+              <FaTrash onClick={clickDelete} />
```

Comment:
it looks like there is a button, and inside a ?FontAwesome Icon?. 
the onClick is on the Icon, which is unusual, since there is a button that would be perfect to use the onClick there. Is there a reason to do it like this?